demonstrations/tutorial_mps.py

```diff
@@ -0,0 +1,679 @@
+================================================================
+
+Matrix Product States are the workhorse for a broad range of modern classical quantum simulation techniques,
+still to this day. Their unique features like offering a canonical form make them incredibly neat to handle 
+in terms of simplicity and algorithmic complexity.
+In this demo, we are going to cover all the essentials you need to know in order to use them for quantum simulation.
+
+
+.. figure:: ../_static/demo_thumbnails/opengraph_demo_thumbnails/OGthumbnail_shadow_hamiltonian_simulation.png
+    :align: center
+    :width: 70%
+
+    
+Introduction
+------------
+
+Matrix Product States (MPS) are an efficient representation of constant-entangled states in one spatial dimension. 
+But in practice, due to their unique features, they are employed in a variety of tasks beyond just gapped 1D systems.
+
+The amount of entanglement the MPS can represent is user-controlled via a hyper-parameter, the so-called `bond dimension` :math:`\chi`.
+If we allow :math:`\chi` to be of :math:`\mathcal{O}(2^{\frac{n}{2}})` for a system of :math:`n` qubits, we can write `any` state as an `exact` MPS.
+To avoid exponentially large resources, however, one typically sets a finite bond dimension :math:`\chi` at the cost of introducing an approximation error.
+
+For some specific classes of states, this is provably sufficient to have a faithful representations (see section on entanglement). 
+But because MPS come with a lot of powerful computational features that we are going to discuss later (in particular canonical forms),
+they are still used in much more complex systems where these requirements do not hold anymore, and still yield good results.
+For example, state of the art quantum chemistry simulations were performed using MPS [#Baiardi]_ 
+and similar methods have been used to simulate experiments on the largest available quantum computers at the time [#Patra]_.
+
+It is known that there are more suitable tensor network states for 
+more complex situations (see section on entanglement later).
+However, they all suffer from significantly more complicated algorithms 
+and higher costs as a lot of the things that make MPS so attractive, 
+like the availability of a canonical form, are not true anymore.
+To put it plainly, it is often simply much easier to use readily available 
+MPS code and throw a large amount of resources into the bond dimension than to develop
+more advanced tensor network methods.
+
+An exception to that are so-called `simple update`
+methods [#Jiang]_, that use pseudo canonical-forms that allow a similarly simple
+algorithmic complexity at the cast of not being optimal in its resources.
+Reference [#Patra]_ is a good example for that.
+
+More advanced tensor network methods are developed and optimized, though the biggest hindrance
+on the widespread use of already known advanced tensor network methods is the lack of reliable open source implementations.
```

Comment:
```suggestion
More advanced tensor network methods are developed and optimized, though the biggest hindrance to the widespread use of already known advanced tensor network methods is the lack of reliable open-source implementations.
```
Grammarly suggests that the correct form is 'hindrance to' instead of 'hindrance on', but I am not sure since I am not an English-native speaker (feel free to ignore if this is not the case)